src/main/java/org/truffleruby/core/encoding/EncodingConverterNodes.java

```diff
@@ -60,6 +60,7 @@
 import java.util.Map;
 import java.util.Map.Entry;
 
+import static org.truffleruby.core.rope.CodeRange.CR_UNKNOWN;
```

Comment:
Do we use static imports elsewhere?

Reply:
Yes. Usually when we're reusing the enum values a lot and it'd reduce code length.